pornhub/video.py

```diff
@@ -0,0 +1,120 @@
+            "pornstars"         : None,     # list
+            "categories"        : None,     # list
+            "tags"              : None,     # list
+            "productions"       : None,     # list
+            "img_url"           : None,     # string
+            "embed_url"         : None      # string
+        }
+
+        # Scrap duration, upload_date, author, img_url, embed_url, accurate_views
+        try:
+            script_data = self._scrapScriptInfo(soup_data.find("script", type='application/ld+json').text)
+        except:
+            data['title'] = '***Video not available in your country***'
+            return data
+
+        for key in script_data:
+            data[key] = script_data[key]
+        
+
+        video = soup_data.find("div", class_='video-wrapper')
+        data['title'] = video.find("span", class_='inlineFree').text
+        data['views'] = video.find("span", class_="count").text  # Scrap view
+        data['rating'] = int(video.find("span", class_="percent").text.replace('%',''))  # Scrap rating
+        data["loaded"] = video.find("div", class_="videoInfo").text  # Scrap loaded
+        data["like"] = video.find("span", class_="votesUp").text  # Scrap like
+        data["accurate_like"] = video.find("span", class_="votesUp")["data-rating"]  # Scrap accurate_like
+        data["dislike"] = video.find("span", class_="votesDown").text  # Scrap dislike
+        data["accurate_dislike"] = video.find("span", class_="votesDown")["data-rating"] # Scrap accurate_dislike
+        data["favorite"] = video.find("span", class_="favoritesCounter").text.strip() # Scrap favorite
+
+        # Scrap pornstars
+        pornstars = [] 
+        for star in video.find_all('a', class_='pstar-list-btn'):
+            pornstars.append(star.text.strip())
+        data["pornstars"] = pornstars
+        
+        # Scrap categories
+        categories = []
+        for category in video.find("div", class_="categoriesWrapper").find_all('a', class_="item"):
+            categories.append(category.text)
+        data["categories"] = categories
+
+        # Scrap tags
+        tags = []
+        for tag in video.find("div", class_="tagsWrapper").find_all('a', class_="item"):
+            tags.append(tag.text)
+        data["tags"] = tags
+
+        # Scrap productions
+        productions = []
+        for prod in video.find("div", class_="productionWrapper").find_all('a', class_="item"):
+            productions.append(prod.text)
+        data["productions"] = productions
+
+        return data
+
+    def getVideo(self, url=None, viewkey=None, *args):
+        """
+        Get video informations.
+        You can enter the full video url or just the viewkey
+        :url: video url on phub
+        :viewkey: viewkey of video
+        """
+        if url or viewkey:
+            data = self._scrapInfo(self._loadPage(url, viewkey))
+        else:
+            return print('URL or Viewkey not entered')
+        return data
```

Comment:
Don't return `data` here because it is only defined in ln 116. Also, the line above, in the `else` block is already returning, so this is quite a mixed logic.